io.github.denzilferreira.asteroid_shooter.json

```diff
@@ -32,13 +32,13 @@
       "name": "asteroid-shooter",
       "buildsystem": "simple",
       "build-commands": [
-        "bash ./build.sh"
+        "bash ./build.sh --prefix=${FLATPAK_DEST}"
```

Comment:
I have no idea what you are trying to do here.

Reply:
was testing to see if this was something to do with where the flatpak was being installed on the CICD and not being able to find the .desktop file (the android vs asteroid :see_no_evil: )